src/Paprika/Data/SlottedArray.cs

```diff
@@ -316,6 +316,16 @@ private void Deframent()
         Debug.Assert(copy._header.Deleted == 0, "All deleted should be gone");
     }
 
+    public void DeleteByPrefix(in NibblePath key)
```

Comment:
Nice and tidy.

Reply:
Anything I need to add here? Or it's just a comment XD.